File: c++/1066.cpp
#include <iostream>

using namespace std;

int main() {
    int positivos=0, negativos=0, pares=0, impares=0;
    int Ns[5], i;

    for (i=0;i<5;i++) {
        cin >> Ns[i];
    }
    for (i=0;i<5;i++) {
        if (Ns[i] > 0) {
            positivos += 1;
        } else if (Ns[i] < 0) {
            negativos += 1;
        }
        if (Ns[i] % 2 == 0) {
            pares += 1;
        } else {
            impares += 1;
        }
    }

    cout << pares << " valor(es) par(es)\n";
    cout << impares << " valor(es) impar(es)\n";
    cout << positivos << " valor(es) positivo(s)\n";
    cout << negativos << " valor(es) negativo(s)\n";
    return 0;
}

File: categorias/iniciante/python/1066.py
# -*- coding: utf-8 -*-

i = pa = p = n = 0
for x in range(5):
    tmp = int(input())
    if tmp%2==0:
        pa += 1
    if tmp%2!=0:
        i += 1
    if tmp>0:
        p += 1
    if tmp<0:
        n += 1

print('''{} valor(es) par(es)
{} valor(es) impar(es)
{} valor(es) positivo(s)
{} valor(es) negativo(s)'''.format(pa, i, p, n))
